hypertrace-trace-enricher/enriched-span-constants/build.gradle.kts

```diff
@@ -63,7 +63,8 @@ sourceSets {
 }
 
 dependencies {
-  api("com.google.protobuf:protobuf-java-util:3.23.3")
+  api(platform("org.hypertrace.core.kafkastreams.framework:kafka-bom:0.2.14"))
```

Comment:
I don't think we want the bom in the constants - they're not using kafka (I hope!) and that means we're dictating a kafka version to any consumer of the constants.

Reply:
Reverting this.

Reply:
Done

Reply:
Done